spec/models/project_spec.rb

```diff
@@ -300,6 +300,22 @@
         end
       end
     end
+
+    context "a hidden project that is active" do
+      let!(:project) { Project.create(platform: "NPM", name: "react", status: "Hidden") }
```

Comment:
```suggestion
      let!(:project) { create(:project, :npm, name: "react", status: "Hidden") }
```
Why not use the factory here?